libbeat/docs/command-reference.asciidoc

```diff
@@ -524,20 +547,18 @@ the end of the file is reached. By default harvesters are closed after
 endif::[]
 
 *`--setup`*::
-ifdef::deprecate_dashboard_loading[]
-deprecated[{deprecate_dashboard_loading}]
-endif::[]
-+
 ifdef::has_ml_jobs[]
 Loads the initial setup, including Elasticsearch template, {kib} index pattern,
 {kib} dashboards (when available), and Machine learning jobs.
-endif::[]
-ifndef::has_ml_jobs[]
+endif::has_ml_jobs[]
+ifdef::no_dashboards[]
```

Comment:
Good find. I need to create another case for that scenario...

Reply:
Oh man, this turns into a real mess with four possibilities:
```
ifdef::has_ml_jobs[]
ifdef::no_dashboards[]
Loads the initial setup, including Elasticsearch template and Machine learning jobs.
endif::no_dashboards[]
ifndef::no_dashboards[]
Loads the initial setup, including Elasticsearch template, {kib} index pattern,
{kib} dashboards (when available), and Machine learning jobs.
endif::no_dashboards[]
endif::has_ml_jobs[]
ifndef::has_ml_jobs[]
ifdef::no_dashboards[]
Loads the initial setup, including Elasticsearch template.
endif::no_dashboards[]
ifndef::no_dashboards[]
Loads the initial setup, including Elasticsearch template, {kib} index pattern,
and {kib} dashboards (when available).
endif::no_dashboards[]
endif::has_ml_jobs[]
```

Reply:
How do you feel about breaking these up into bullets instead of a sentence? This method would drastically simplify the code:
```
Loads the initial setup, including: 

* Elasticsearch template
ifdef::has_ml_jobs[]
* Machine learning jobs
endif::has_ml_jobs[]
ifndef::no_dashboards[]
* {kib} index pattern
* {kib} dashboards
endif::no_dashboards[]
```

Reply:
GitHub needs a scream emoji. :-) Yah I've done something similar on other pages, so it makes sense here, too.